modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -89,6 +93,48 @@ finished.
 > the codemod, as its resulting formatting (spacing, quotes, etc.) may not match your project
 > conventions.
 
+## Component Promotions
+
+### Table (Preview)
+
+**PR:** [#2600](https://github.com/Workday/canvas-kit/pull/2600)
+
+We've promoted the `Table` component in preview to main.
+
+## Removals
+
+### Table (Main)
+
+**PR:** [#2600](https://github.com/Workday/canvas-kit/pull/2600)
+
+We've removed the `Table` component in main and replaced it with the `Table` component from preview.
+This will consist of a new API since it is now a compound component.
+
+> `rowState` prop is no longer a part of the `Table` component.
+
+See below for how these components will migrate:
+
+|      v10       |         v11        |
+| -------------- | ------------------ |
+| `<Table/>`     | `<Table/>`         |
+|      `n/a`     | `<Table.Caption/>` |
+|      `n/a`     | `<Table.Head/>`    |
+|      `n/a`     | `<Table.Body/>`    |
+| `<TableRow/>`  | `<Table.Row/>`     |
+|      `n/a`     | `<Table.Header/>`  |
+|      `n/a`     | `<Table.Cell/>`    |
+|      `n/a`     | `<Table.Footer/>`  |
+
+In v10, `Table` and `TableRow` were the pieces that would be used to build a Table.
+In v11, `Table` is now a compound component that utilizes all the pieces of an html
+[Table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table) element to build a Table.
+
+
```

Comment:
Suggestion: Seems like we should have a basic example of the former Table and how to implement it in v11 with the new Table. Or we should link out to a GH discussion.